include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -58,18 +58,15 @@ class format_parse : public format_base
     format_parse & operator=(format_parse const & pf) = default; //!< Defaulted.
     format_parse(format_parse &&) = default;                     //!< Defaulted.
     format_parse & operator=(format_parse &&) = default;         //!< Defaulted.
+    ~format_parse() = default;                                   //!< Defaulted.
 
     /*!\brief The constructor of the parse format.
      * \param[in] argc_ The number of command line arguments.
      * \param[in] argv_ The command line arguments to parse.
      */
-    format_parse(int const argc_, char const * const * const  argv_) :
-        argc(argc_ - 1)
-    {
-        init(argc_, argv_);
-    }
-
-    ~format_parse() = default;
+    format_parse(int const argc_, std::vector<std::string> && argv_) :
```

Comment:
```suggestion
    format_parse(int const argc_, std::vector<std::string_view> && argv_) :
```

Reply:
I need a vector of strings in format_parse because I alter the contents. For example, a cluster of boolean flags `-fgh` is evaluated by stripping each flag once I have queried it from the cluster such that I can still identify which flags are set and which do not exist.

Reply:
Hm, I would argue that the function that needs copies should create the copies, but it's not so important.